examples/cryptocurrency-advanced/backend/src/api.rs

```diff
@@ -50,28 +50,33 @@ pub struct WalletProof {
 /// Wallet history.
 #[derive(Debug, Serialize, Deserialize)]
 pub struct WalletHistory {
+    /// Proof to the transaction list.
```

Comment:
Not transaction list, but transaction hashes list (or list of transaction hashes)